cub/cub/util_math.cuh

```diff
@@ -78,15 +79,13 @@ _CCCL_HOST_DEVICE _CCCL_FORCEINLINE OffsetT safe_add_bound_to_max(OffsetT lhs, O
  * `(n + d - 1)` would overflow.
  */
 template <typename NumeratorT, typename DenominatorT>
+CUB_DEPRECATED_BECAUSE("Use cuda::ceil_div instead")
 _CCCL_HOST_DEVICE _CCCL_FORCEINLINE constexpr NumeratorT DivideAndRoundUp(NumeratorT n, DenominatorT d)
```

Comment:
Important: Please add a comment that this function is deprecated since CCCL 2.8.